JSchugt/HomePRO
# Js client side materials

# Description
Registration working on fire base
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
From login screen select register, fill in data and submit,  user should now have an account
# Checklist:
- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
